yarn-project/pxe/src/pxe_service/pxe_service.ts

```diff
@@ -205,9 +203,11 @@ export class PXEService implements PXE {
     const contractDaos = contracts.map(c => new ContractDao(c.artifact, c.completeAddress, c.portalContract));
```

Comment:
I'm wondering if we should now enqueue the contents of this function using the job queue?

Reply:
Yes for sure. Thanks!